osmaxx/api_client/API_client.py

```diff
@@ -90,5 +91,28 @@ def _authorization_headers(self):
         }
 
 
+class LazyChunkedRemoteFile:
+    """Wrapper around to-be-downloaded files
+
+    Can be passed to django.core.files.storage.Storage#save as ``content``
+    if that storage's save logic consumes chunks like ``FileSystemStorage`` does
+    at https://github.com/django/django/blob/1.9.4/django/core/files/storage.py#L252
+
+    Notes:
+        This is not a file-like object: It has no ``read`` method.
```

Comment:
A `read` method could probably provided by delegating to [`response.raw.read()`](http://docs.python-requests.org/en/v2.8.1/api/#requests.Response.raw), but I haven't tested that and for now YAGNI.